infra/build/developer-tools-light/build/install_cloud_sdk.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
```

Comment:
@bharathkkb We shouldn't copy this file, instead either source directly from https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/install_cloud_sdk.sh or COPY the file over from the other image.

Reply:
So we cannot source directly since it is not available in the build context (different dir) and in the other image we [remove](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/2ddc9147cd8f94be3d64910b7dd1d737da96e6cb/infra/build/developer-tools/Dockerfile#L173) `/build`. A workaround could be in the makefile, we can copy it over at buildtime, but that would mean if a user tried to build from the directory directly `developer-tools-light` instead of makefile it will fail.

Reply:
Should we move the build context to be in the parent directory instead then?

Reply:
@morgante done, the light image now builds at `/infra/build`. Note this still means if user runs docker build inside `developer-tools-light` they will run into an error.